<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FruitDialog</class>
 <widget class="QDialog" name="FruitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>新鲜的水果！！！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在前面打钩就可以了</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>721</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clearPushButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="returnPushButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="payPushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="appleSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bananaSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="waterSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pearSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="papayaCheckBox_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>木瓜</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="muguaSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="papayaCheckBox_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>350</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>木瓜</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="peachSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/new/prefix0/resource/shuiguo.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="waterlemonCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>西瓜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="pearCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>梨子</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="muguaCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>木瓜</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="appleCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>苹果</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="banalaCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>香蕉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="peachCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>桃子</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>clearPushButton</zorder>
  <zorder>returnPushButton</zorder>
  <zorder>appleSpinBox</zorder>
  <zorder>bananaSpinBox</zorder>
  <zorder>waterSpinBox</zorder>
  <zorder>pearSpinBox</zorder>
  <zorder>papayaCheckBox_2</zorder>
  <zorder>muguaSpinBox</zorder>
  <zorder>papayaCheckBox_3</zorder>
  <zorder>peachSpinBox</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
